recipes-azure-iot/iot-hub-device-update/iot-hub-device-update/swupdate_handler_v2_grub.sh

```diff
@@ -513,6 +602,39 @@ CheckIsInstalledState() {
     result "$aduc_result"
 }
 
+#
+# Implementation of 'IsInstallable' function, for the reference Yocto Image.
```

Comment:
what reference Yocto Image? ... this is intended for an omnect-os image (same finding for other script)

Reply:
done